.github/workflows/ci.yml

```diff
@@ -9,11 +9,11 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: set up python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
-          python-version: 3.9
+          python-version: 3.9 
```

Comment:
Please configure your IDE to remove trailing whitespaces on save and remove this one here

Reply:
There is a `re-run workflow` button in the GitHub UI that will do the same thing without needing to push a commit. Only time we should need to push a commit is when GitHub Actions is down and decides to not re-queue our builds.

Reply:
The button wasn't there, so I had pretty much to push an empty commit to re-run the Github Actions.